lib/awful/widget/calendar_popup.lua

```diff
@@ -246,9 +246,12 @@ end
 --
 -- @param widget Widget to attach the calendar
 -- @tparam[opt="tr"] string position Two characters string defining the position on the screen
+-- @tparam[opt={}] table args Additional options
+-- @tparam[opt=false] bool args.on_hover Show popup while mouse hover
```

Comment:
I've not yet used it myself.
What about clicking makes it stay visible until you click it again?
Is that the case already?